organization/_posts/2018-05-24-coordination.md

```diff
@@ -0,0 +1,154 @@
+================
+
+Licensing
+---------
+-   See [attached documents](https://indico.cern.ch/event/722019/).
+-   We need to follow up with FastJet.
+
+Training
+--------
+-   Discussion with Sudhir. He volunteered to host a web page on the
+    Fermilab platform with links to existing training schools and
+    material. In the beginning just a web page, but will grow into an
+    effective portal for training. Will happen at the end of next
+    week, when he will be done with teaching duties.
+
+Packaging
+---------
+-   Will contribute to the [LIM
+    workshop](https://indico.cern.ch/event/720948/) next
+    week. Next meeting [13
+    June](https://indico.cern.ch/event/730538/).
+    -   aliBuild and Portage contributions to the test drive are
+        currently PRs.
+    -   Ben worked on a
+        [guide](https://github.com/HSF/packaging/tree/master/istools)
+        to working with different instruction sets.
+
+Frameworks
+----------
+-   [BOF at CHEP](https://indico.cern.ch/event/727646/)
+    -   Talks semi-confirmed from Kyle Knoepfel, Chris Jones, Alex
+        Buchanan, Holger Schulz, and Alexander Souza. Speakers
+        expected from ALICE and ATLAS.
+    -   Indico to be filled in soon; will add a registration page to
+        help planning.
+    -   Focus on I/O and data models.
+
+Visualization
+-------------
+-   WebEventDisplay project is being setup so that people can
+    collaborate on a common WebGL based event display (this was
+    initially developed by Ed Moyse and is used for the TrackML
+    challenge).
+
+Software Development
+--------------------
+-   Martin, Giulio and Graeme met and we will have a Software
+    Development meeting on 13th June (taking this 15-16h slot). Aim is
+    to make progress and identify people wanting to work on
+    performance & optimisation topics. Also proposing a BOF follow up
+    at CHEP.
+
+HSF/WLCG Workshop Follow-up
+===========================
+-   Practical follow ups at the HSF level
+    -   Inventory of community activities
+        -   [http://www.hepsoftware.org/](http://www.hepsoftware.org/)
+            is dormant - do we need a new "technology" (for example, a
+            graph database may look as an attractive technology)? But
+            re-use the information for sure. Link this to Sudhir's
+            work with training inventory perhaps?
+        - Does not preclude a quick reorganisation of the website with
+          more highlighting of the more active areas.
+    -   More variation in meeting themes (more topical):
+        -   DD4hep meeting confirmed for 6 June.
+        -   Do we restart this in the Software Tech Forum?
+            -   Torre: that never seemed to work too well - rethink this
+                activity.
+            -   Links to HSF were weaker than they should be.
+    -   Collaboration guide for projects
+        -   Draft by Benedikt. Iterating with Graeme.
+-   Are we missing the big picture for "software upgrades"?
```

Comment:
I would emphasize a bit more that this was the big discussion this time. Don't have a good idea how though